duckduckgo/Android
# Update privacy grade so a score of 2 or 3 is now a C.

Asana Issue URL: https://app.asana.com/0/488551667048375/530378792792821

## Description
Update privacy grade so a score of 2 or 3 is now a C. Also add unit tests for letter grade (score is already tested) now that it is not a direct mapping 

## Steps to Test this PR:
1. Visit whitehouse.gov
1. Open privacy dashboard
1. Ensure that the before score is a 'C' not a 'D'
